core/src/edu/asu/c3simulator/util/PlayerInterfaceImplementation.java

```diff
@@ -0,0 +1,89 @@
+package edu.asu.c3simulator.util;
+
+import edu.asu.c3simulator.simulation.C3Simulation;
+import edu.asu.c3simulator.simulation.Player;
+import edu.asu.c3simulator.simulation.SocioeconomicStatus;
+
+/**
+ * Implementation of the Player interface Provides detail about various player
+ * attributes that are needed for PlayerStatusDisplay
+ * 
+ * @author Reigel, Justin
+ *
+ */
+
+public class PlayerInterfaceImplementation implements Player
+{
+	private SocioeconomicStatus status;
+	private int netWorth;
+	private int capital;
+	private C3Simulation simulation;
+
+	public PlayerInterfaceImplementation(SocioeconomicStatus newStatus, int newNetWorth,
+			int newCapital, C3Simulation newSimulation)
+	{
+		status = newStatus;
+		netWorth = newNetWorth;
+		capital = newCapital;
+		simulation = newSimulation;
+	}
+
+	/**
+	 * @return The net worth of the player in base units
+	 */
+	@Override
+	public int getNetWorth()
+	{
+		// TODO Auto-generated method stub
+		return netWorth;
+	}
+
+	/**
+	 * @return The amount of money the player owns as capital
+	 */
+	@Override
+	public int getCapital()
+	{
+		// TODO Auto-generated method stub
+		return capital;
+	}
+
+	/**
+	 * @return Parent simulation to which this player belongs
+	 */
+	@Override
```

Comment:
Just a note: our styleguide does not require javadocs for methods with the @Override annotation, but they are acceptable if they add something new.

Also note: If these comments are identical to the comments in the interface that they override, then they should be removed. If a method has an @Override annotation but no javadoc, it reverts to its parent's documentation (which makes maintaining the documentation much easier).

Copy + Paste = Bad +- 0.3